.github/workflows/ubuntu-wheel.yml

```diff
@@ -67,13 +67,17 @@ jobs:
           elif [ "${{ env.PYTHON_VERSION }}" = "3.10" ] && [ "${{ env.DEVELOPER_BUILD }}" = "OFF" ]; then
             docker/docker_build.sh cuda_wheel_py310
```

Comment:
Questions:

1. Although CPU wheel is created as part of the CUDA build, shall we use one of the `./docker_build cpu-xxx` options to build CPU wheel for CI?
2. Regardless of the decision of 1, if we call `./docker_build cpu-static`, shall it create a `open3d.whl` or `open3d-cpu.whl`?

Reply:
1. That would need compiling a lot of code twice - once for the full wheel and once for the CPU wheel. The current CI process reuses the code compiled for the full wheel.
2. `-cpu` is added to the created wheel whenever `BUILD_CUDE_MODULE==OFF` and on Linux x86_64 (see `setup.py`) So any docker / host build will add that.